src/data_model/eng2math.jl

```diff
@@ -0,0 +1,811 @@
+import LinearAlgebra: diagm
+
+"items that are mapped one-to-one from engineering to math models"
+const _1to1_maps = Dict{String,Vector{String}}(
+    "bus" => ["vm", "va", "terminals", "phases", "neutral", "dss"],
```

Comment:
should include also `"vm_pn_lb", "vm_pn_ub", "vm_pp_lb", "vm_pp_ub", "vm_ng_ub", "vm_pair_ub", "vm_pair_lb"`

Reply:
You can leave out `"vm_pair_ub"` and `"vm_pair_lb"`because this wouldn't work with the pu conversion in any case. I will add this in a minor release later